app/models/course/assessment/question/text_response_solution.rb

```diff
@@ -0,0 +1,9 @@
+class Course::Assessment::Question::TextResponseSolution < ActiveRecord::Base
+  enum solution_type: [:exact_match, :keyword]
+
+  # validate grade <= max grade?
```

Comment:
Added actual validation instead. Was meant as TODO (so `# TODO`: in the future?)